lib/src/modal_style.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class ElTooltipModalStyle {
```

Comment:
Hi @hellohejinyu 
Can you just change the name of the file to reflect the name of the class?
It should be `el_tooltip_modal_style.dart`
Thanks
Gil